File: Add Code Here/JAVA/Atm.java
20.	import java.util.Scanner;  
21.	public class Atm
22.	{  
23.	     public static void main(String args[] )  
24.	    {  
25.	              int balance = 100000, withdraw, deposit;  
26.	          
27.	            Scanner sc = new Scanner(System.in);  
28.	          
29.	        while(true)  
30.	        {  
31.	            System.out.println("Automated Teller Machine");  
32.	            System.out.println("Choose 1 for Withdraw");  
33.	            System.out.println("Choose 2 for Deposit");  
34.	            System.out.println("Choose 3 for Check Balance");  
35.	            System.out.println("Choose 4 for EXIT");  
36.	            System.out.print("Choose the operation you want to perform:");  
37.	              
38.	                     int choice = sc.nextInt();  
39.	            switch(choice)  
40.	            {  
41.	                case 1:  
42.	        System.out.print("Enter money to be withdrawn:");  
43.	                      
44.	              withdraw = sc.nextInt();           
45.	        if(balance >= withdraw)  
46.	        {  
47.	                   balance = balance - withdraw;  
48.	            System.out.println("Please collect your money");  
49.	        }  
50.	        else  
51.	        {  
52.	                      System.out.println("Insufficient Balance");  
53.	        }  
54.	        System.out.println("");  
55.	        break;  
56.	   
57.	                case 2:  
58.	                      
59.	        System.out.print("Enter money to be deposited:");  
60.	                      
61.	             deposit = sc.nextInt();  
62.	                      
63.	               balance = balance + deposit;  
64.	        System.out.println("Your Money has been successfully depsited");  
65.	        System.out.println("");  
66.	        break;  
67.	   
68.	                case 3:   
69.	        System.out.println("Balance : "+balance);  
70.	        System.out.println("");  
71.	        break;  
72.	   
73.	                case 4:  
74.	        System.exit(0);  
75.	            }  
76.	        }  
77.	    }  
78.	}  

File: Add Code Here/PYTHON/ATM.py
class Atm:

    counter=0
    def __init__(self):
        self.__pin = ""
        self.__balance = 0
        self.sno = Atm.counter
        Atm.counter+=1
        
        self.menu()
    
    def get_pin(self):
        return self.__pin
    
    def set_pin(self,new_pin):
        if type(new_pin) == str:
            self.__pin = new_pin
            print("Pin changed")
        else:
            print("Not allowed")

    def menu(self):
        user_input = input("""HEllo how would like to proceed ?\n1. Enter 1 to create pin\n2. Enter 2 to deposit\n3. Enter 3 to withdraw\n4. Enter 4 to check balance\n5. Enter 5 to exit\n""")
        if user_input == '1':
            self.create_pin()
        elif user_input == '2':
            self.deposit()
        elif user_input == '3':
            self.withdraw()
        elif user_input == '1':
            self.check_balance()
        else:
            print("Bye")

    def create_pin(self):
        self.__pin = input("Enter your pin : ")
        print("Pin set successfully")
    def deposit(self):
        temp = input("Enter your pin : ")
        if temp==self.__pin:
            amount = int(input("Enter your amount"))
            self.__balance += amount
            print("Deposit succesfull")
        else:
            print("Invalid pin")

    def withdraw(self):
        temp = input("Enter your pin : ")
        if temp==self.__pin:
            amount = int(input("Enter your amount"))
            if amount<=self.__balance:
                self.__balance -= amount
                print("Operation succesfull")
            else:
                print("Insufficent funds")
        else:
            print("Invalid pin")
    
    def check_balance(self):
        temp = input("Enter your pin : ")
        if temp==self.__pin:
            print("Balance :",self.__balance)
        else:
            print("Invalid pin")

sbi = Atm()
sbi.deposit()
sbi.check_balance()
sbi.withdraw()